### ISSUE TITLE
How to use opentypse.js and poly2tri.js within Jupyter notebook

### ISSUE THREAD

**Issue by khbilen** [jupyter/notebook#1500]
I am trying to use opentype.js and poly2tri.js with Jupyter notebook, along with my own additional JS scripts. I copied all these to AppData/Roaming/Jupyter/nbextensions folder . enabled them from tthe console. I can see the notebook.JSON file at ./jupyter folder with load_extensions: { "scriptxx": true} for each of them. 
For a simplified demo, I collected all other JS scripts within the tryText.html file. This html file executes properly on the browser. But if I execute it on a notebook cell via the IPython.display.HTML command, I get " Reference error: opentype is not defined". Same is true, if I try to run poly2tri.js if I run it with manually provided data (i.e., without opentype.js provided data). As I understand, the notebook can not see these packages ( although I do not get 404 error on the server console, but 302 error; which I get for my own local scripts that still work OK)
How can I get these two working with Jupyter notebook?
[tryText.zip](https://github.com/jupyter/notebook/files/290187/tryText.zip)

**Comment by takluyver** [comment]
It's hard to follow exactly what you're trying without the files in question, but I think you might be expecting nbextensions to run in the global namespace. They aren't: each nbextension is loaded in a separate namespace.

You could define a module in each nbextension and then require it in your display object. Or it may be simpler to display all the Javascript rather than putting it into an nbextension.

**Comment by khbilen** [comment]
Thanks for your response. Sorry for my ignorance, I thought I had zipped everyting in a file. I hope now all in one. opentype needs a subdirectory called fonts, with the Roboto font in it. I've added them al togther. I am trying to use plain JS as much as possible, and JS only as much as possible; but I think these two arein factory structure, somehow notebook does not recongize them. 
[tryText.zip](https://github.com/jupyter/notebook/files/291699/tryText.zip)

**Comment by jtpio** [comment]
Closing since the classic notebook UI has now been moved to the `nbclassic` repository: https://github.com/jupyter/nbclassic

If you think this issue is still relevant, please open a new one there, thanks!
